ceremonies/src/benchmarking.rs

```diff
@@ -257,6 +257,35 @@ benchmarks! {
 		assert_eq!(AttestationCount::<T>::get((cid, cindex)), 1);
 	}
 
+	attest_attendees {
+		let cid = create_community::<T>();
+
+		let attestor = generate_pair();
+		let attestor_account = account_id::<T>(&attestor);
+
+		assert_ok!(Pallet::<T>::register_participant(
+			RawOrigin::Signed(attestor_account.clone()).into(),
+			cid,
+			Some(fake_last_attendance_and_get_proof::<T>(&attestor, cid)))
+		);
+
+		let attestees =  register_users::<T>(cid, 2, 7).into_iter().map(|u| account_id::<T>(&u)).collect();
+
```

Comment:
To ensure that we test the worst-case scenario, we should increase the meetup size a bit. Maybe up to 15?

Reply:
i tried, but then the participants are split over 2 meetups by the assignment algo